bcipy/signal/process/transform.py

```diff
@@ -29,6 +30,23 @@ def __call__(self, data: np.ndarray, fs: Optional[int] = None) -> Tuple[np.ndarr
             return data[:, :: self.factor], None
 
 
+class TransformParams(NamedTuple):
```

Comment:
`DefaultP300TransformParams` or `ERPTransformParams`? We might have some other defaults depending on the application.

Reply:
I agree with Tab on this